### PR TITLE
Dw2360-3 Introduction to Github assignment

### PR DESCRIPTION
**Pull request by dw2360**
Introduction to GitHub assignment changes based on JJm's comments.

### CODE REVIEW

File: .vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (bjones1) [inline .vscode/c_cpp_properties.json:1]: Delete all these files from your PR.

File: thorgame/dw2360/power_of_thor.py [hunk 2]
```diff
@@ -50,12 +55,17 @@
 
     # Write an action using print To debug: print("Debug messages...",
     # file=sys.stderr, flush=True)
-    if initial_ty < light_y & initial_tx < light_x:
+
+    # Compares Thor's initial position with the position of the light of power.  
+    # If Thor's y coordinates are less than the light of power's y coordinates, 
+    # and if Thor's x coordinates are less than the light of power's y coordinates,   
+    # then add 1 unit to Thor's x and y position, thus moving SouthEast until Thor reaches the light of power
+    if initial_ty < light_y and initial_tx < light_x:
```
Comment (bjones1) [inline thorgame/dw2360/power_of_thor.py:63]: This is an incomplete solution -- it works, but doesn't handle moving in directions other than SE, SW, W. Add cases for the other directions for completeness.

**Review by bjones1 — CHANGES_REQUESTED** [review]

**Comment by bjones1** [comment]
I don't see any updates to this PR.

**Comment by dw2360** [comment]
Ready to Review.

**Pull request merged**